bazel/external/rules_rust.patch

```diff
@@ -0,0 +1,15 @@
+# https://github.com/bazelbuild/rules_rust/pull/1315
+
+diff --git a/rust/private/rustc.bzl b/rust/private/rustc.bzl
+index 6cdbefeb..284d4afa 100644
+--- a/rust/private/rustc.bzl
++++ b/rust/private/rustc.bzl
+@@ -1024,7 +1024,7 @@ def rustc_compile_action(
+         ),
+     ]
+ 
+-    if crate_info.type in ["staticlib", "cdylib"]:
++    if crate_info.type in ["staticlib", "cdylib"] and not out_binary:
```

Comment:
Just to understand the context, do you expect this to eventually become unnecessary once https://github.com/bazelbuild/rules_rust/issues/771 is resolved?

Reply:
Even once https://github.com/bazelbuild/rules_rust/pull/1315 is resolved, since it worked in the past, but they broke it during refactoring and now refuse to merge this one-line fix...

Reply:
huh that's unfortunate...